tss-esapi/src/structures/buffers/private.rs

```diff
@@ -7,9 +7,36 @@ use crate::{
 };
 use std::convert::TryInto;
 use tss_esapi_sys::_PRIVATE;
+use serde::{Deserialize, Deserializer, Serialize, Serializer};
 
 buffer_type!(Private, ::std::mem::size_of::<_PRIVATE>(), TPM2B_PRIVATE);
 
+impl Serialize for Private {
+    fn serialize<S>(&self, serializer: S) -> std::result::Result<S::Ok, S::Error>
+    where
+        S: Serializer,
+    {
+        let bytes = self.marshall()
+            .map_err(|err| {
+                serde::ser::Error::custom(err)
```

Comment:
Hmmm... annoying codespell... you'll need to add `ser` to `.codespellrc`

Reply:
Will do :)